lib/include/tree_sitter/parser.h

```diff
@@ -62,13 +62,13 @@ typedef struct {
       TSStateId state;
       bool extra : 1;
       bool repetition : 1;
-    };
+    } status;
```

Comment:
I think this field should be called `shift`. "Shift" and "reduce" are the two main types of actions.

Reply:
yeah sure. I didn't know how to name it as I'm not that familiar with tree-sitter internals